exporter/awsemfexporter/metric_translator.go

```diff
@@ -215,22 +220,30 @@ func getCWMetrics(metric *pdata.Metric, namespace string, instrumentationLibName
 		dps = DoubleDataPointSlice{metric.DoubleSum().DataPoints()}
 	case pdata.MetricDataTypeDoubleHistogram:
 		dps = DoubleHistogramDataPointSlice{metric.DoubleHistogram().DataPoints()}
+	default:
+		config.logger.Warn(
```

Comment:
`.debug` level I think, this will be too spammy right?

Reply:
Hey @anuraaga, thanks for the feedback! Since this is an edge case, it shouldn't happen often, and we do want to alert the customer that this is happening vs just having it in the debug level. I'm happy to turn this down to `debug` if down the road this edge case is occurring more frequently than we think. :) 

cc: @mxiamxia

Reply:
Please fix this in a followup PR if needed